util.py

```diff
@@ -72,3 +73,14 @@ def ifindex_to_batiface(if_index, batman_ifaces):
     if iface in batman_ifaces or iface == None:
         return iface
     return iface_match_recursive(iface, batman_ifaces)
+
+def read_domainfile(dcf_path):
+    """Read a json file which holds all currently known assignments of
+       bat interfaces to domains as a dictionary and return it as python dict.
+       Return an empty dict, if the given path was None.
+    """
+    if dcf_path is None:
+        return {}
+    with open(dcf_path, "r") as dc_file:
+        known = json.load(dc_file)
+        return known
```

Comment:
```suggestion
        retorn json.load(dc_file)

```

Reply:
Thought about logging at that point, but later found that that's not a thing in mesh-announce, yet.